test/common/ssl/context_impl_test.cc

```diff
@@ -343,19 +343,19 @@ TEST(ClientContextConfigImplTest, EmptyServerNameIndication) {
                             EnvoyException, "SNI names containing NULL-byte are not allowed");
 }
 
-// Multiple certificate hashes are not yet supported.
-// TODO(htuch): Support multiple hashes.
-TEST(ClientContextConfigImplTest, MultipleValidationHashes) {
+// Validate hex-encoded SHA-256.
```

Comment:
This comment doesn't really describe what the test does.

Reply:
Fixed.